### PR TITLE
scale the blank slate image, so its visible on larger displays

### LINKED ISSUE #1946: Blank slates are really small on hi-res displays
### Description

When running on a hi-res display (in this instance it's 4K with Windows set to 150% scaling) the blank slate illustrations are really small.

### Version
**GitHub Desktop version:** 0.6.0 (has existed since they were added)
**OS version:** Win10 CU

### Steps to Reproduce

1. Buy a 4k display
1. Set Windows to 150% scaling
1. 💰

**Expected behaviour:** The illustrations should scale so on hi-res displays they appear larger

**Actual behaviour:** The illustrations don't scale and are really small

**Reproduces how often:** Every time

#### Additional Information

This is a screen shot of my desktop with the application at the approx. size I run it at.

desktop

### PR DESCRIPTION
**Pull request by shimpossible**
makes the slate a % of the screen so its visible.
fixes #1946 

:heart:

### CODE REVIEW

**Comment by joshaber** [comment]
@niik could you review this? 🙏

**Comment by zhouhaoyang** [comment]
@shimpossible 

hi, I see you create a pull request for 'desktop' :)

May I ask is there documentation that teach us how to build windows release from 'desktop' source code ? 

So I can edit and build my own release?

Thank you very much!

File: app/styles/ui/_blank-slate.scss [hunk 1]
```diff
@@ -57,5 +57,6 @@
 
 .blankslate-image {
   margin: var(--spacing-double) auto;
-  width: 400px;
+  min-width: 400px;
```
Comment (niik) [inline app/styles/ui/_blank-slate.scss]: Hey @shimpossible, sorry for taking so long on this. Could you please add a `max-width: 800px` here as well so we have some constraints on it?

Comment (shimpossible) [inline app/styles/ui/_blank-slate.scss]: Is there a reason you picked 800?

Comment (niik) [inline app/styles/ui/_blank-slate.scss]: Yeah, it's 2x the minimum width which is a reasonable scale boundary.

Comment (shimpossible) [inline app/styles/ui/_blank-slate.scss]: I've not tested this on a 4k screen, only 2k.   But 800 is probably reasonable

**Comment by shimpossible** [comment]
🍨

**Comment by niik** [comment]
Thanks @shimpossible!

**Pull request merged**